source/cost-benefits-of-pay.html.erb

```diff
@@ -0,0 +1,138 @@
+            <li class="sub-navigation__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a class="govuk-link" href="/payment-service-provider" itemprop="item"><span itemprop="name">GOV.UK Pay’s Payment Service Provider</span></a>
+            </li>
+            <li class="sub-navigation__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a class="govuk-link" href="/direct-debit" itemprop="item"><span itemprop="name">Direct Debit</span></a>
+            </li>
+            <li class="sub-navigation__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a class="govuk-link" href="/apple-pay-and-google-pay" itemprop="item"><span itemprop="name">Apple Pay and Google Pay</span></a>
+            </li>
+            <li class="sub-navigation__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a class="govuk-link" href="/roadmap" itemprop="item"><span itemprop="name">Roadmap</span></a>
+            </li>
+            <li class="sub-navigation__item" itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem">
+              <a class="govuk-link" href="/security" itemprop="item"><span itemprop="name">Security and compliance</span></a>
+            </li>
+          </ol>
+        </nav>
+      </div>
+
+      <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">The cost benefits of using GOV.UK Pay</h1>
+
+      <p class="govuk-body">GOV.UK Pay is designed, built and tested by government to meet the needs of the public sector and its users.</p>
+      <p class="govuk-body">To help explain the costs of using Pay, we’ve set out what you do and do not pay for.</p>
+
+      <h2 class="govuk-heading-m">What you pay for</h2>
+      <p class="govuk-body">The Pay platform is free to use. You only pay transaction fees to your Payment Service Provider (PSP). These vary depending on the card type used and whether you choose: </p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>GOV.UK Pay’s contract with our PSP Stripe</li>
+        <li>Government Banking’s contract with Worldpay</li>
+      </ul>
+
+      <p class="govuk-body">To learn about Stripe’s fees, <a class="govuk-link" href="https://selfservice.payments.service.gov.uk/create-service/register">create a Pay account</a> or <a class="govuk-link" href="https://selfservice.payments.service.gov.uk/login">sign in</a>. (It’s free to do this and there is no contractual commitment.) In the footer, choose either: </p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>GOV.UK Pay memorandum of understanding for Crown bodies</li>
+        <li>Contract for non-crown bodies</li>
+      </ul>
+
+      <p class="govuk-body">For Government Banking’s Worldpay fees, email <a class="govuk-link" href="mailto:serviceteam.gbs@hmrc.gov.uk">serviceteam.gbs@hmrc.gov.uk.</a></p>
```

Comment:
I think the full stop should be after the </a> close tag.

Reply:
Done.